sql/executor.go

```diff
@@ -285,6 +286,8 @@ func (p parameters) Arg(name string) (parser.Datum, bool) {
 		return parser.DString(t.StringVal), true
 	case *driver.Datum_TimeVal:
 		return parser.DTimestamp{Time: t.TimeVal.GoTime().UTC()}, true
+	case *driver.Datum_IntervalVal:
```

Comment:
good point, added. it's a shame we can't exercise this since the Go driver will never send `Datum_DateVal`.